tests/test_utils.py

```diff
@@ -75,9 +75,9 @@ def test_to_laplace_RDAD(self):
 
     def test_to_laplace_regularizer_kwarg(self):
         expected_L_normed = [
-            [0, 1 / sqrt(6), 0],
-            [1 / sqrt(6), 0, 1 / sqrt(6)],
-            [0, 1 / sqrt(6), 0],
+            [0, 3 / sqrt(70), 0],
```

Comment:
why did this change?

Reply:
The previous test was not passing and considering the R-DAD default regularizer is 1, this test with the regularizer equal to 1 should give the same output as R-DAD with no specified regularizer.